readme.md

```diff
@@ -1,4 +1,9 @@
-# XMLDOM [![Build Status](https://secure.travis-ci.org/xmldom/xmldom.png?branch=master)](http://travis-ci.org/xmldom/xmldom) [![NPM version](https://badge.fury.io/js/xmldom.png)](http://badge.fury.io/js/xmldom)
+# XMLDOM
+
+[![license](https://img.shields.io/npm/l/xmldom?color=blue&style=flat-square)](./LICENSE.md)
+[![npm](https://img.shields.io/npm/v/xmldom?style=flat-square)](https://www.npmjs.com/package/xmldom)
+[![bug issues](https://img.shields.io/github/issues/xmldom/xmldom/bug?color=red&style=flat-square)](https://github.com/xmldom/xmldom/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+[!["help wanted" issues](https://img.shields.io/github/issues/xmldom/xmldom/help%20wanted?color=darkgreen&style=flat-square)](https://github.com/xmldom/xmldom/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
```

Comment:
One step further, maybe off-topic, would be to use a dash in the "help wanted" issues label, which I think would be just as clear and would make the hyperlinking a little easier in general. In the past I noticed that renaming an issues label would break old links, not sure if this is still the case or not.

Reply:
That change would have been easier to do before landing this.
I would assume that the links still break, but I don't know for sure.

Reply:
Yeah I was a little too quick to land the "help wanted" badge. I think not super tragic, though.